src/main/java/com/woowacamp/soolsool/core/statistics/dto/response/LiquorSalePriceResponse.java

```diff
@@ -0,0 +1,29 @@
+package com.woowacamp.soolsool.core.statistics.dto.response;
+
+import com.woowacamp.soolsool.core.liquor.domain.Liquor;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+public class LiquorSalePriceResponse {
+
+    private Long id;
+    private String name;
+    private String brew;
+    private String region;
+    private String imageUrl;
+    private Long totalSalePrice;
```

Comment:
```suggestion
@Getter
@RequiredArgsConstructor
public class LiquorSalePriceResponse {

    private final Long id;
    private final String name;
    private final String brew;
    private final String region;
    private final String imageUrl;
    private final Long totalSalePrice;
```

지금까지 DTO 패턴과 일관성을 맞춰봤습니다!